src/fetch/payouts.py

```diff
@@ -321,17 +344,18 @@ def construct_payout_dataframe(
     # TODO - After CIP-20 phased in, adapt query to return `solver` like all the others
     slippage_df = slippage_df.rename(columns={"solver_address": "solver"})
     # 1. Assert existence of required columns.
-    validate_df_columns(payment_df, slippage_df, reward_target_df)
+    validate_df_columns(payment_df, slippage_df, reward_target_df, mev_kickback_df)
 
     # 2. Normalize Join Column (and Ethereum Address Field)
     join_column = "solver"
-    normalize_address_field(payment_df, join_column)
-    normalize_address_field(slippage_df, join_column)
-    normalize_address_field(reward_target_df, join_column)
```

Comment:
Note that a slight logical change was made here... instead of calling normalize address on each data-frame individually we iterate over a loop of data-frames and call the method. This is "mutating" method so its not entirely clear to me if it does what we expect when "borrowing" the entries of an array and mutating them. Certainly the way that it has been written implies that I expect it to work the same. I might write a little test example to show that this is indeed the case.

Reply:
Like this. Good work, Ben.